app/models/spree/back_in_stock_notification.rb

```diff
@@ -1,4 +1,6 @@
 class Spree::BackInStockNotification < ApplicationRecord
+  require 'csv'
```

Comment:
Usually `require` calls go at the top of the file, not inside of a class, though functionally it doesn't matter.

Reply:
Thanks I have updated that